audio-worklet/src/bitcrusher-demo.js

```diff
@@ -85,26 +123,65 @@ class BitcrusherDemo {
           default: this.masterGain_.gain.value,
           name: 'Volume'
         });
+
+    // GUI controls allow switching between audio worklet and script processor.
+    // The script processor node is used by default, and switching is not
+    // enabled if the browser does not support audio worklet.
+    if (this.useWorkletAndScriptProcessor_) {
+      this.useScriptProcessorCheckBox_ = document.createElement('input');
+      this.useScriptProcessorCheckBox_.type = 'checkbox';
+      this.useScriptProcessorCheckBox_.checked = true;
+      const audioworkletLabel = document.createElement('label');
+      audioworkletLabel.textContent =
+          'Use ScriptProcessor (if unchecked, use AudioWorklet Bitcrusher).';
+
+      this.useScriptProcessorCheckBox_.addEventListener(
+          'change', this.switchBitcrusherNode.bind(this));
+
+      let container = document.getElementById(containerId);
+      container.appendChild(this.useScriptProcessorCheckBox_);
+      container.appendChild(audioworkletLabel);
+    }
+  }
+
+  /**
+   * Switch between whether the audio worklet node or script processor node
+   * connects to the master gain node.
+   */
+  switchBitcrusherNode() {
+    // The function is called after the checkbox switches state.
+    const changeToAudioWorklet = !this.useScriptProcessorCheckBox_.checked;
+    if (changeToAudioWorklet) {
+      this.bitcrusherAudioWorklet_.connect(this.masterGain_);
+      this.bitcrusherScriptProcessor_.output.disconnect();
+    } else {
+      this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
+      this.bitcrusherAudioWorklet_.disconnect();
+    }
```

Comment:
With current design, the checkbox is only added to the GUI if audio worklet is implemented in the browser, so the method is never called.